integrations/auth0/oauth.go

```diff
@@ -70,38 +70,12 @@ func (h handler) handleOAuth(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	// Tests OAuth0's Management API.
-	req, err := http.NewRequestWithContext(r.Context(), http.MethodGet, fmt.Sprintf("https://%s/api/v2/roles", d), nil)
-	if err != nil {
-		l.Error("Failed to create HTTP request", zap.Error(err))
-		c.AbortServerError("request creation error")
-		return
-	}
-
-	req.Header.Set("Authorization", "Bearer "+oauthToken.AccessToken)
-
-	client := &http.Client{}
-	resp, err := client.Do(req)
-	if err != nil {
-		l.Error("Failed to execute HTTP request", zap.Error(err))
-		c.AbortBadRequest("execution error")
-		return
-	}
-	defer resp.Body.Close()
-
-	body, err := io.ReadAll(resp.Body)
-	if err != nil {
-		l.Error("Failed to read response body", zap.Error(err))
-		c.AbortServerError("response reading error")
-		return
-	}
-
-	// Check response status code.
-	if resp.StatusCode != http.StatusOK {
-		l.Error("Auth0 API request failed",
-			zap.Int("status_code", resp.StatusCode),
-			zap.String("response", string(body)))
-		c.AbortBadRequest("Auth0 API request failed")
+	// Tests Auth0's Management API.
+	url := fmt.Sprintf("https://%s/api/v2/roles", d)
+	auth := "Bearer " + oauthToken.AccessToken
```

Comment:
This is coming around a log, maybe a `GenAuth` function

Reply:
We have a common function for hadling OAuth tokens being used in HTTP requests: `FreshOAuthToken`.

As for prepending `"Bearer "`: while it's a common and standard-based pattern, unfortunately it's not absolute, and replacing `a + b` with `a(b)` actually obscures the meaning, even if it's simple.